packages/intrepid2/src/Cell/Intrepid2_CellDataDef.hpp

```diff
@@ -821,7 +821,85 @@ refCenterDataStatic_ = {
     // wedge
     { 1.0/3.0, 1.0/3.0, 0.0},
 };
-}
+
+
+// Point Inclusion 
+
+
+  template<typename PointViewType>
+  KOKKOS_INLINE_FUNCTION
+  bool
+  PointInclusion<shards::Line<>::key>::
+  check(const PointViewType &point, const double threshold) {
+    const double minus_one = -1.0 - threshold, plus_one = 1.0 + threshold;
+    return (minus_one <= point(0) && point(0) <= plus_one);
+  }  
+
+  template<typename PointViewType>
+  KOKKOS_INLINE_FUNCTION
+  bool
+  PointInclusion<shards::Triangle<>::key>::
+  check(const PointViewType &point, const double threshold) {
+    const double distance = max( max( -point(0), -point(1) ), point(0) + point(1) - 1.0 );
```

Comment:
I think this causes a compilation error when the incoming `PointViewType` uses `float` instead of `double`. @alanw0 @mperego

Reply:
(In particular, this code stub here: https://github.com/trilinos/Trilinos/blob/2ad26029e5e3592de143b1640480382a296c401a/packages/stk/stk_unit_tests/stk_search_util/UnitTestIsInElement.cpp#L160)

Reply:
Thanks @MalachiTimothyPhillips, would it help if that `check` function used `typename PointViewType::value_type` instead of `double`?

Reply:
Thanks @MalachiTimothyPhillips, PR #13394 should fix this.